packaging/release.py

```diff
@@ -0,0 +1,1410 @@
+
+    commit = get_commit(commit)
+    version = get_ansible_version(commit=commit)
+    tag = f"v{version}"
+
+    if upstream_tag := git("ls-remote", "--tags", upstream.name, tag, capture_output=True).stdout.strip():
+        with suppress_when(allow_tag):
+            raise ApplicationError(f"Version {version} has already been tagged: {upstream_tag}")
+
+    upstream_branch = get_upstream_branch(version)
+    upstream_refs = git("branch", "-r", "--format=%(refname)", "--contains", commit, capture_output=True).stdout.strip().splitlines()
+    upstream_ref = f"refs/remotes/{upstream.name}/{upstream_branch}"
+
+    if upstream_ref not in upstream_refs:
+        raise ApplicationError(f"Commit {upstream_ref} not found. Found {len(upstream_refs)} upstream ref(s): {', '.join(upstream_refs)}")
+
+    body = create_github_release_notes(upstream, repository, version, validate)
+
+    release = GitHubRelease(
+        user=upstream.user,
+        repo=upstream.repo,
+        target=commit,
+        tag=tag,
+        title=tag,
+        body=body,
+        pre_release=version.pre is not None,
+    )
+
+    create_github_release(release)
+
+
+@command
+def post_version() -> None:
+    """Set the post release version."""
+    current_version = get_ansible_version()
+    requested_version = get_ansible_version(f"{current_version}.post0", mode=VersionMode.REQUIRE_POST)
+
+    set_ansible_version(current_version, requested_version)
+
+
+@command
+def create_post_pr(allow_stale: bool = False) -> None:
+    """Create a branch and open a browser tab for creating a post release pull request."""
+    version = get_ansible_version(mode=VersionMode.REQUIRE_POST)
+
+    pr = prepare_pull_request(
+        version=version,
+        branch=f"release-{version}-{secrets.token_hex(4)}",
+        title=f"Update Ansible release version to v{version}.",
+        add=(ANSIBLE_RELEASE_FILE,),
+        allow_stale=allow_stale,
+    )
+
+    create_pull_request(pr)
+
+
+@command
+def release_announcement(repository: str, version: str | None = None, mailto: bool = True, validate: bool = True) -> None:
+    """Generate a release announcement for the current or specified version."""
+    parsed_version = get_ansible_version(version, mode=VersionMode.STRIP_POST)
+    upstream = get_remotes().upstream
+    message = create_release_announcement(upstream, repository, parsed_version, validate)
+    recipients = ", ".join(RELEASE_ANNOUNCEMENT_RECIPIENTS)
+
+    if mailto:
+        to = urllib.parse.quote(recipients)
+
+        params = dict(
+            subject=message.subject,
+            body=message.body,
+        )
+
+        query_string = urllib.parse.urlencode(params)
+        url = f"mailto:{to}?{query_string}"
+
+        display.show("Opening email client through default web browser ...")
+        webbrowser.open(url)
+    else:
+        print(f"TO: {recipients}")
+        print(f"SUBJECT: {message.subject}\n{message.body}")
```

Comment:
How about having a visual empty line?
```suggestion
        print(f"SUBJECT: {message.subject}\n\n{message.body}")
```

Reply:
There's a blank line before the body now.